### PR TITLE
Adding cisco spark notifier plugin

### PR DESCRIPTION
**Pull request by taylorp36**

### CODE REVIEW

File: permissions/plugin-cisco-spark-notifier.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "cisco-spark-notifier"
+paths:
+- "org/jenkins-ci/plugins/cisco-spark-notifier"
```
Comment (taylorp36) [inline permissions/plugin-cisco-spark-notifier.yml]: Woops sorry about that, fixed it here:
https://github.com/jenkinsci/cisco-spark-notifier-plugin/blob/master/build.gradle#L32

**Comment by taylorp36** [comment]
Woops sorry about that, fixed it here:
https://github.com/jenkinsci/cisco-spark-notifier-plugin/blob/master/build.gradle#L32

**Comment by batmat** [comment]
:+1:

**Comment by daniel-beck** [comment]
@taylorp36 Merged, thanks!

**Pull request merged**